### DISCUSSION TITLE
Cant make any changes to the svelte files

### DISCUSSION THREAD

**Discussion by Sandbird** [vkurko/calendar#402] [Q&A] [answered]
I've followed the guide here: https://github.com/vkurko/calendar/discussions/59, everything is working fine and see the demo site on my browser.
Now, any changes that i make in packages\resource-timeline\src\Sidebar.svelte file , like adding a simple test class:
![image](https://github.com/user-attachments/assets/5897b0ce-19f1-405f-96bb-1b69cab82115)

when i run 'npm run build' (--force as well), even though i see the change made in index.js
![image](https://github.com/user-attachments/assets/652da808-558c-4419-a5cf-de2c4f21ff32)

the final event-calendar.min.js does not include the change.
I've tried everything there is to try, (npm cache clear --force, npm run clean etc), removed and reinstalled everything from scratch, tried a npm update, everything i can think of, nothing helped.

When i am running npm run dev, i can see the changes in the console, but still the final js does not include them:
![image](https://github.com/user-attachments/assets/9d4457d8-16fd-475b-ab57-4a97cc972060)

Any ideas?

**Comment by vkurko** [comment, marked as answer]
Can you please check your `node-modules/@event-calendar` directory? There should be symlinks like this:

```shell
$ ls -l node_modules/@event-calendar/

build -> ../../packages/build/
core -> ../../packages/core/
day-grid -> ../../packages/day-grid/
interaction -> ../../packages/interaction/
list -> ../../packages/list/
resource-time-grid -> ../../packages/resource-time-grid/
resource-timeline -> ../../packages/resource-timeline/
time-grid -> ../../packages/time-grid/
```

If this is not the case, then try the following:

1. Go to every package in `packages` directory and run `npm link`:
```shell
cd packages/build
npm link
cd ../core
npm link
...
(and so on)
```

2. Empty `node_modules/@event-calendar` directory:
```shell
rm -rf node_modules/@event-calendar/*
```

3. In the project's directory run:
```shell
npm link @event-calendar/build @event-calendar/core ... (and so on)
```

**Comment by Sandbird** [comment]
Hmm, I thought it was a linking problem. I am using Windows 10 here, so my guess is this is a windows issue. The 'npm link @event-calendar/build' part, didnt work as expected. Each line was just creating a link for that specific folder, but it was undoing the previous symlink changes from the previous commands. So basically in the end I was left with just the 'time-grid' folder link.
I managed to set up all links with [JunctionMaster](https://bitsum.com/portfolio/junctionmaster/).
Thanks, it is working now

**Reply by vkurko** [reply]
> Each line was just creating a link for that specific folder, but it was undoing the previous symlink changes from the previous commands.

You're right. The correct option is to do everything in one command:

```shell
npm link @event-calendar/build @event-calendar/core ... (and so on)
```

**Comment by Sandbird** [comment]
Uh, good to know, thanks
